### DISCUSSION TITLE
1.1 Search by age - asumir edad válida

### DISCUSSION THREAD

**Discussion by MystEdu** [IIC2133-PUC/2024-2#136] [Tarea 2] [answered]
Buenas, en este evento

![Captura de pantalla 2024-10-15 182232](https://github.com/user-attachments/assets/a05bb2a2-baba-4935-b0c4-10a26a95fb7a)

¿Se puede asumir que siempre existe al menos 1 pingüino con la edad entregada?

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, si pueden asumirlo.

**Reply by blampasona18** [reply]
Hola podemos asumir esto para las 4 consultas de la primera parte? Es decir, siempre habrá al menos un penguin en el rango de pesos y en las áreas de rectángulo y círculo?

**Reply by sebacarrascop** [reply]
Hola!, si, recien actualizamos los tests porque no estaba pasando con el in-circle, pero con la nueva versión si pasa :)
